newIDE/app/src/EventsSheet/EventsTree/Renderers/JsCodeEvent.js

```diff
@@ -140,6 +140,7 @@ export default class JsCodeEvent extends React.Component<
           [selectableArea]: true,
         })}
         onClick={this.editObject}
+        style={this.props.disabled ? styles.comment : {}}
```

Comment:
nit: can you factor all the styles into a variable: `const textStyle = this.props.disabled ? styles.comment : undefined`

(you don't need to pass an empty object, you can set the style to undefined if you have nothing to set).